pipes/WDL/tasks/tasks_nextstrain.wdl

```diff
@@ -307,6 +309,8 @@ task export_auspice_json {
             --metadata ~{metadata} \
             $NODE_DATA_FLAG ~{sep=' ' select_all([branch_lengths,nt_muts,aa_muts])}\
             --auspice-config ~{auspice_config} \
+            ${"--lat-longs" + lat_longs_tsv} \
```

Comment:
Should we be consistent about `~{}` vs `${}`, at least per-file? The [WDL spec](https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md#expression-placeholders) says `~{}` is preferred now (though I still like`${}`).

Reply:
true -- I'm new to this notation, and also didn't realize it applied to these other situations (with prefixes before the substitutions, or even the `sep`) but we should stay consistent at least witihin the task level